obi/pyobi/tests/test_pyobi.py

```diff
@@ -2,7 +2,48 @@
 from pyobi import *
 
 
-def test_encode_signed_integer():
+def test_encode_decode_bool():
+    # encode
+    assert PyObiBool().encode(True) == bytes.fromhex("01")
+    assert PyObiBool().encode(False) == bytes.fromhex("00")
+
+    # decode
+    assert PyObiBool().decode(bytes.fromhex("01")) == (True, b"")
+    assert PyObiBool().decode(bytes.fromhex("00")) == (False, b"")
+
```

Comment:
For each type test add decode test that has left bytes after decode. Like assert PyObiBool().decode(bytes.fromhex("00aa")) == (False, bytes.fromhex("aa"))

Reply:
done